app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,283 @@
+</p>
+
+<p> 
+    Some of the Account API methods are available from Server SDKs when you authenticate with a JWT. This allows you to perform actions on behalf of a project account's user on the backend.
+</p>
+
+<h2><a href="/docs/authentication#create-an-account" id="create-an-account">Create An Account</a></h2>
+
+<p>
+    A user account in Appwrite is the primary way to access information for a given project. Accounts can be created in many different ways, including <a href="/docs/client/account#accountCreate">email & password</a>, <a href="/docs/client/account#accountCreateAnonymousSession">anonymous sessions</a>, <a href="/docs/client/account#accountCreateOAuth2Session">OAuth2</a>, <a href="/docs/client/account#accountCreatePhoneSession">phone authentication</a>, and more. Applications can create and manage authentication sessions through the REST API or Client SDKs.
+</p>
+
+<h3><a href="/docs/authentication#email" id="email">Email</a></h3>
+
+<p>
+    Creating an account via email and password is one of the most common ways to sign up for an application. Appwrite provides email and password authentication out of the box. Using one of Appwrite's Client SDKs, or the REST APIs directly, you can create an account using an email address and password in your application.
+</p>
+
+<p>
+    Passwords are hashed with <a href="https://en.wikipedia.org/wiki/Argon2" target="_blank" rel="noopener">Argon2</a>, a resilient and secure password hashing algorithm.
+</p>
+
+<p>
+    The example below shows you how to create an account using the Web SDK:
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.create('[USER_ID]', 'email@example.com', 'password');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});</code></pre>
+</div>
+
+<p>
+    After an account is created, the newly created account can be verified through the <a href="/docs/account#accountCreateVerification">account verification route</a> provided by the Appwrite Accounts API. The user doesn't need to be verified to log in, but you can restrict resource access to verified users only using permissions.
+</p>
+
+<h3><a href="/docs/authentication#anonymous-user" id="anonymous-user">Anonymous User</a></h3>
+
+<p>
+    Anonymous authentication allows users of your application to create a temporary valid session without creating an account. The session will last for a year. If an account is created with an active anonymous session, it will be attached to the existing anonymous session.
```

Comment:
```suggestion
    Anonymous authentication allows users of your application to create a temporary valid session without creating an account. The session has an expiry of one year. If an account is created while an anonymous session is active, it will be attached to the existing anonymous session.
```